src/main.jsx

```diff
@@ -39,6 +39,15 @@ const Layout = () => {
     await deleteDoc(docRef);
   }
 
+  async function updatePlace(e, editedPlace) {
```

Comment:
I notice this is in main.jsx. It'd probably be advantageous to move functions that edit data into a separate file for ease of understanding.